engine/src/flutter/impeller/renderer/backend/gles/shader_library_gles.cc

```diff
@@ -67,12 +67,12 @@ ShaderLibraryGLES::ShaderLibraryGLES(
     return true;
   };
   for (auto library : shader_libraries) {
-    auto blob_library = ShaderArchive{std::move(library)};
-    if (!blob_library.IsValid()) {
+    auto blob_library = ShaderArchive::Create(std::move(library));
+    if (!blob_library.ok()) {
```

Comment:
We should probably be reporting the status here, but we already have a PR open to go back and surface these to dart.